src/gengowatcher/web.py

```
@@ -965,7 +974,12 @@ async def health_check():
             "timestamp": time.time(),
         }
     except Exception as e:
-        return {"status": "unhealthy", "detail": str(e), "timestamp": time.time()}
+        api_instance.logger.exception(f"Health check failed: {e}")
```

Ruff (0.14.14)

Redundant exception object included in `logging.exception` call

(TRY401)